spec/sidekiq/form1010cg/submission_job_spec.rb

```diff
@@ -66,17 +65,115 @@
     end
   end
 
-  describe 'when job has failed' do
+  describe 'when retries are exhausted' do
+    after do
+      Flipper.disable(:caregiver_use_va_notify_on_submission_failure)
```

Comment:
🤔 my understanding is that Flipper flags are enabled by default - should we set them back to enabled in the `after`?

Reply:
Yeah, that's probably the right call. It's backwards to how I would do it/think of it, but if you look in the config/initializers/flipper.rb we explicitly turn them all on in test environments. I'll update it 👍